Entity/UploadedAsset.php

```diff
@@ -261,7 +261,7 @@ public function getType()
     /**
      * Set user
      *
-     * @param string $user
+     * @param User $user
```

Comment:
Nope it's a string, there is non hard link between the ems core data model and a user entity. Always a soft username string.

Reply:
we should probably rename this variable to username

Reply:
Ok I Will take care of this. I see Indeed that doctrine mapping is string. I Will change the @var of the property